pkgs/top-level/aliases.nix

```diff
@@ -455,6 +455,8 @@ mapAliases ({
   links = links2; # added 2016-01-31
   linux_rpi0 = linuxKernel.kernels.linux_rpi1;
   linuxPackages_rpi0 = linuxKernel.packages.linux_rpi1;
+  linux_rpi02w = linuxKernel.kernels.linux_rpi3;
+  linuxPackages_rpi02w = linuxKernel.packages.linux_rpi3;
```

Comment:
I don't know if we should add this alias.

Reply:
This was for consistency with `linux_rpi0`, which was added in #68265.

Reply:
I still don't know, I don't care that much, it just might get confusing, the more device-specific kernel attrnames we accumulate.

Ideally I'd remove all vendor kernels from NIxpkgs!

Reply:
I think we should add it for consistency, and so that it's more straightforward for users.